dev/TabView/TabView.idl

```diff
@@ -7,6 +7,16 @@ enum TabViewWidthMode
 {
     Equal = 0,
     SizeToContent = 1,
+    Compact = 2,
```

Comment:
>Compact [](start = 4, length = 7)

This value is being added to a public enum, has it gone though API review?

Reply:
@ranjeshj @stmoy

Reply:
Working on getting it through API review: https://github.com/microsoft/microsoft-ui-xaml-specs/pull/70/files

Reply:
It should be possible to add [WUXC_VERSION_PREVIEW] just above the new enum entry to unblock this checkin. @chingucoding can you give that a try ?

Reply:
I think I have tried that and the compiler wasn't happy with it. It seemed like adding an attribute here was not allowed in an enum.

Reply:
I see. It is fine then. It is not a breaking change since it is only additive and hopefully @stmoy can take this through API review before we release the stable release in which case this becomes a non-issue.